internal/templates/templates.go

```diff
@@ -148,6 +148,71 @@ func GetExportTemplates() ([]ExportTemplate, error) {
 
 }
 
+func GetReportTemplates() ([]ReportTemplate, error) {
+	var reportTemplates []ReportTemplate
+	var templates []ReportTemplate
+	home, err := os.UserHomeDir()
+	if err != nil {
+		return nil, err
+	}
+
+	templateFiles := []string{filepath.Join(home, ".sailpoint", "report-templates.json")}
+
+	customTemplates := config.GetCustomReportTemplatePath()
+	if customTemplates != "" {
+		templateFiles = append(templateFiles, customTemplates)
+	}
+
+	envSearchTemplates := os.Getenv("SAIL_REPORT_TEMPLATES_PATH")
+	if envSearchTemplates != "" {
+		templateFiles = append(templateFiles, envSearchTemplates)
+	}
+
+	for i := 0; i < len(templateFiles); i++ {
+		templateFile := templateFiles[i]
+
+		file, err := os.OpenFile(templateFile, os.O_RDWR, 0777)
+		if err != nil {
+			if config.GetDebug() {
```

Comment:
I wonder if we should have a global wrapper for log statements that does this debug check? Otherwise we are going to be checking this everywhere.

Reply:
I will eventually be migrating all of these kinds of statements to the debug function on our primary log package, and that will solve those issues